features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin/source/src/app/components/Workflow/APIStateChange.jsx

```diff
@@ -97,10 +91,22 @@ function ListLabels() {
         return restApi
             .workflowsGet('AM_API_STATE')
             .then((result) => {
-                const workflowlist = result.body.list;
-                return (workflowlist);
+                const workflowlist = result.body.list.map((obj) => {
+                    return {
+                        description: obj.description,
+                        api: obj.properties.apiName + '-' + obj.properties.apiVersion,
+                        action: obj.properties.action,
+                        currentState: obj.properties.currentState,
+                        apiProvider: obj.properties.currentState,
+                        referenceId: obj.referenceId,
+                        createdTime: obj.createdTime,
+                        properties: obj.properties,
+                    };
+                });
+                return workflowlist;
             })
             .catch((error) => {
+                Alert.error('Unable to get workflow pending requests for API State Change', error.message);
```

Comment:
Shall we Internationalise the static message?

Reply:
fixed with 9c884f8